frontend/src/pages/projects/utils.ts

```diff
@@ -48,3 +66,136 @@ export const getSecretDisplayName = (secret: SecretKind): string =>
   getDisplayNameFromK8sResource(secret);
 export const getSecretDescription = (secret: SecretKind): string =>
   getDescriptionFromK8sResource(secret);
+
+export const typedBorderColor = (objectType: ProjectObjectType): string => {
+  switch (objectType) {
+    case ProjectObjectType.project:
+      return 'var(--ai-project--BorderColor)';
```

Comment:
This is a weird separation of concern point -- I'd like the variables not to be exposed, but we can't really talk ProjectObjectType... maybe an enum for the types based on the "ai-<type>"?